scripts/signature.py

```diff
@@ -169,7 +171,7 @@ def parse_args(args):
     command = parser.add_subparsers(dest="command")
     export_command_parser = command.add_parser("export")
     export_command_parser.add_argument("--spark", action="store_true")
-    export_command_parser.add_argument("--presto", action="store_false")
+    export_command_parser.add_argument("--presto", action="store_true")
```

Comment:
If I am not mistaken , store_false means presto defaults to true if no flag of spark presto is specified right ? 
Our default was to have presto enabled by default and thus the store_false, which is probably why register_presto_signatures gets called even with spark enabled. Thus I think just changing store_false to store_true aught to be enough. 

(Also If we would like both spark and presto signatures in one registry  , calling clear signatures would not work.)